assembly.py

```diff
@@ -471,6 +472,52 @@ def order_and_orient(inFasta, inReference, outFasta,
     #    os.unlink(trimmed)
     return 0
 
+def _call_order_and_orient_orig(kwargs):
+    _order_and_orient_orig(**kwargs)
+
+def order_and_orient(inFasta, inReference, outFasta,
+        outAlternateContigs=None, outReference=None,
+        breaklen=None, # aligner='nucmer', circular=False, trimmed_contigs=None,
```

Comment:
Are these used?

Reply:
args are passed through to _order_and_orient_orig()